src/extend-expect.ts

```diff
@@ -1,6 +1,4 @@
-// eslint-disable-next-line @typescript-eslint/triple-slash-reference, spaced-comment
-/// <reference path="../jest.d.ts" />
-// eslint-disable-next-line import/namespace
+import {expect} from "@jest/globals";
```

Comment:
I am wondering if this would break support for those who don't use the globals import. Or maybe more likely, break support with the Jest versions that don't have a globals import in the core jest package.